### ISSUE TITLE
Identity 

### ISSUE THREAD

**Issue by kathytpham** [kathytpham/prime-user-experience#6]
Since the UI wasn’t originally working, I tried three different Stanford Accounts. I couldn’t tell right away whose data this was until I started digging in the results.
![img_7407](https://cloud.githubusercontent.com/assets/10030313/8842343/bbc21462-30ad-11e5-870b-816835e01270.PNG)

**Comment by thetylerhayes** [comment]
To be clear, when you say 3 different accounts, these are 3 different people? E.g., you don't have 3 different accounts all for yourself in Stanford right? ;)

**Comment by kathytpham** [comment]
Yes, three different people. :) I currently manage three different people's accounts at Stanford alone. :)
